lib/extended-css.js

```diff
@@ -150,102 +182,50 @@ var ExtendedCss = function (styleSheet) { // jshint ignore:line
      * @param rules Rules to apply
      */
     var applyRules = function (rules) {
+        // var start = performance.now();
```

Comment:
I guess that's the right place to record timings.

What we need:

1. The total number of times applyRules was called
2. Total time spent
3. Avg time per call
4. Std deviation

On a side note, it sometimes useful to see if the page is slow due to the ExtCss rules. Do you see any simple way to enable timings recording from the extension side?